doc/installation_and_administration_guide/configuration.md

```diff
@@ -310,7 +312,61 @@ cygnusagent.channels.mkafka-channel.transactionCapacity = 100
 
 [Top](#top)
 
-##<a name="section4"></a>`log4j.properties`
+##<a name="section4"></a>`flume-env.sh`
+The file `flume-env.sh` can be instantiated from a template given in the Cygnus repository, `conf/flume-env.sh.template`. 
+
+```
+#=============================================
+# To be put in APACHE_FLUME_HOME/conf/flume-env.sh
+#=============================================
+
+#JAVA_HOME=/usr/lib/jvm/java-6-sun
+
+# Give Flume more memory and pre-allocate, enable remote monitoring via JMX
+#JAVA_OPTS="-Xms100m -Xmx200m -Dcom.sun.management.jmxremote"
+
+# Note that the Flume conf directory is always included in the classpath.
+#FLUME_CLASSPATH="/path/to/the/flume/classpath"
+```
+
+`flume-env.sh` file has been inherited from Apache Flume, and it is used in order to configure certain Flume parameters such as an alternative classpath, some Java options etc.
+
+[Top](#top)
+
+##<a name="section5"></a>`grouping_rules.conf`
+The file `grouping_rules.conf` can be instantiated from a template given in the Cygnus repository, `conf/grouping_rules.conf.template`. 
+
+
+```
+{
+    "grouping_rules": [
+        {
+            "id": 1,
+            "fields": [
+                ...
+            ],
+            "regex": "...",
+            "destination": "...",
+            "fiware_service_path": "..."
+        },
+        ...
+    ]
+} 
+```
+
+Being:
+
+* <b>id</b>: A unique unsigned integer-based identifier. Not really used in the current implementation, but could be useful in the future.
+* <b>fields</b>: These are the fields that will be concatenated for regular expression matching. The available dictionary of fields for concatenation is "entityId", "entityType" and "servicePath". The order of these fields is important since the concatenation is made from left to right.
+* <b>regex</b>: Java-like regular expression to be applied on the concatenated fields. Special characters like '\' must be escaped ('\' is escaped as "\\\\").
+* <b>destination</b>: Name of the HDFS file or CKAN resource where the data will be effectively persisted. In the case of MySQL, Mongo and STH this sufixes the table/collection name. Please, have a look to [doc/design/naming_conventions.md](doc/design/naming_conventions.md) for more details.
+* <b>fiware\_service\_path</b>: New `fiware-servicePath` replacing the notified one. The sinks will translate this into the name of the HDFS folder or CKAN package where the above destination entity will be placed. In the case of MySQL, Mongo and STH this prefixes the table/collection name. Please, have a look to [doc/design/naming_conventions.md](doc/design/naming_conventions.md) for more details.
+
+`grouping_rules.conf` must be put in `APACHE_FLUME_HOME/conf/`  .
```

Comment:
This line is unnecesary since all the configuration files must be under `conf/`. Anyway, this is something to be remembered in the introduction section of this document.